controller_manager_tests/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(controller_manager_tests)
 
+set_directory_properties(PROPERTIES COMPILE_OPTIONS "-std=c++11")
```

Comment:
For now I've put this here, as it's the only package which actually _requires_ it. Users of controller_interface who don't use `multi_interface_controller.h` don't even need it.

That said, perhaps there should be a separate PR/discussion about doing this across the board.

Reply:
But (existing!) users of `multi_interface_controller.h` need to set this (?).

Reply:
Yes, that's correct. So perhaps this can just be curiosity for now— we can build our system from a fork until the `melodic-devel` branch is created.

Reply:
Since this is now targeting Melodic, I shall remove the compile options.